datascience-notebook/Dockerfile

```diff
@@ -56,4 +75,40 @@ RUN julia -e 'Pkg.add("IJulia")' && \
 # Show Julia where conda libraries are
 # Add essential packages
 RUN echo 'push!(Sys.DL_LOAD_PATH, "/opt/conda/lib")' > /home/$NB_USER/.juliarc.jl && \
-    julia -e 'Pkg.add("Gadfly")' && julia -e 'Pkg.add("RDatasets")' && julia -F -e 'Pkg.add("HDF5")'
+julia -e 'Pkg.add("Gadfly")' && julia -e 'Pkg.add("RDatasets")' && julia -F -e 'Pkg.add("HDF5")'
+
+# Add odbc drivers to the install
+RUN cd /home/jovyan && \
+    wget ftp://ftp.freetds.org/pub/freetds/stable/freetds-patched.tar.gz && \
```

Comment:
Agreed.  Not sure how this would be implemented correctly so accepting of any solution you suggest or code.